src/exception/exceptions.py

```diff
@@ -0,0 +1,29 @@
+from src.flag_evaluation.error_code import ErrorCode
+
+
+class OpenFeatureError(Exception):
+    def __init__(
+        self, error_message: str = None, error_code: ErrorCode = None
+    ):
+        self.error_message = error_message
+        self.error_code = error_code
+
+
+class FlagNotFoundError(OpenFeatureError):
```

Comment:
Not sure if you want to file an issue for it, but we should put docstrings on these.